core/modules/system/system.admin.inc

```diff
@@ -508,6 +508,7 @@ function system_modules($form, $form_state = array()) {
       'autocomplete' => 'off',
       'title' => t('Enter a part of the module name or description to filter.'),
     ),
+    '#default_value' => (backdrop_get_query_parameters()) ? backdrop_get_query_parameters() : '',
```

Comment:
I didn't change it to `search` it was that way already; https://github.com/backdrop/backdrop/blob/1.x/core/modules/system/system.admin.inc#L501

Prolly should stick with that.